### PR TITLE
move a bunch of presets to 4.5

### PR DESCRIPTION
**Pull request by sugaarK**
@limonspb want check with MCK if his rates have changed?

### CODE REVIEW

File: presets/4.3/rc_link/ghost/race.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #$ TITLE: IRC Ghost (Mode: Race)
-#$ FIRMWARE_VERSION: 4.3
+#$ #$ FIRMWARE_VERSION: 4.3
```
Comment (haslinghuis) [inline presets/4.3/rc_link/ghost/race.txt:2]:
```suggestion
#$ FIRMWARE_VERSION: 4.3
```

File: presets/4.3/rc_link/generic/500hz_hd_freestyle.txt [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #$ TITLE: Generic 500Hz HD Freestyle
 #$ FIRMWARE_VERSION: 4.3
 #$ FIRMWARE_VERSION: 4.4
-#$ CATEGORY: RC_LINK
```
Comment (haslinghuis) [inline presets/4.3/rc_link/generic/500hz_hd_freestyle.txt:4]: Seems we need category here

Comment (sugaarK) [reply presets/4.3/rc_link/generic/500hz_hd_freestyle.txt:4]: ohh yeah done

**Comment by limonspb** [comment]
> @limonspb want check with MCK if his rates have changed?

checking with him....

**Comment by limonspb** [comment]
wait with this one plz, let's figure out why @SupaflyFPV  and @spatzengr  want to create new files for 4.5 from scratch

**Comment by sugaarK** [comment]
Any thing wrong here @limonspb ? It all looks good to me

**Review by limonspb — APPROVED** [review]

**Comment by limonspb** [comment]
> Any thing wrong here @limonspb ? It all looks good to me

yes sir looks good

**Pull request merged**